polaris-react/src/components/LegacyCard/LegacyCard.scss

```diff
@@ -135,6 +141,16 @@
     border-top: var(--p-border-width-1) solid var(--p-color-border-subdued);
     margin-top: var(--p-space-5);
   }
+
+  #{$se23} & {
+    border-top: 0;
+  }
+}
+
+.Section-subdued:last-child {
+  #{$se23} & {
+    padding: var(--p-space-4);
+  }
```

Comment:
Ensure subdued footers have even spacing. otherwise the top would have `8px` and it wouldn't look like a footer

<img width="1502" alt="Screenshot 2023-06-22 at 12 40 51 PM" src="https://github.com/Shopify/polaris/assets/20652326/88357264-49d4-4094-96f1-5fc5b79e044e">